src/clr-angular/forms/styles/_checkbox.clarity.scss

```diff
@@ -0,0 +1,114 @@
+        }
+
+        label {
+            @include checkbox-radio-label-styles($clr-forms-checkbox-label-height, $clr-forms-checkbox-padding-left, $clr-forms-checkbox-label-color);
+        }
+
+        //Checkbox base
+        input[type="checkbox"] + label::before {
+            @include checkbox-radio-shared-inactive(
+                $clr-forms-checkbox-height, 
+                $clr-forms-checkbox-top, 
+                0, 
+                $clr-forms-border-color);
+            border-radius: $clr-global-borderradius;
+        }
+
+        input[type="checkbox"]:focus + label::before {
+            @include include-outline-style-form-fields();
+        }
+
+        //Checkmark
+        input[type="checkbox"] + label::after {
+            position: absolute;
+            content: '';
+            display: none;
+            height: $mark-half-size + $clr-rem-1px;
+            width: $clr-forms-checkbox-mark-size;
+            border-left: $clr-forms-checkbox-border-style;
+            border-bottom: $clr-forms-checkbox-border-style;
+            top: ($clr-forms-baseline * 4 - $clr-checkbox-radio-height) / 2;
+            left: $mark-half-size;
+            transform: translate(0, $mark-half-size) rotate(-45deg);
+            border-color: $clr-forms-checkbox-mark-color;
+        }
+
+        // @TODO How to get checked state without a parent class?
+        input[type="checkbox"]:checked + label::before {
+            background: $clr-forms-checkbox-background-color;
+            border: none;
+        }
+
+        input[type="checkbox"]:checked + label::after {
+            display: inline-block;
+        }
+
+        input[type="checkbox"].clr-indeterminate + label::before {
+            border: $clr-default-borderwidth solid $clr-forms-checkbox-indeterminate-border-color;
+        }
+        input[type="checkbox"].clr-indeterminate + label::after {
+            border-left: none;
+            border-bottom-color: $clr-forms-checkbox-indeterminate-border-color;
+            display: inline-block;
+            transform: translate(0, $mark-half-size);
+        }
+
+        label > input[type="checkbox"]::after {
+            position: absolute;
+            content: '';
+            display: none;
+            height: $mark-half-size + $clr-rem-1px;
+            width: $clr-forms-checkbox-mark-size;
+            border-left: $clr-forms-checkbox-border-style;
+            border-bottom: $clr-forms-checkbox-border-style;
+            top: ($clr-forms-baseline * 4 - $clr-forms-checkbox-height) / 2;
+            left: $mark-half-size;
+            transform: translate(0, $checkmark-half-size) rotate(-45deg);
+            border-color: $clr-forms-checkbox-mark-color;
+        }
+
+        &.clr-checkbox-inline {
+            @include form-inline-input();
+        }
+    }
+
+    .clr-form-control-disabled .clr-checkbox-wrapper {
+        input[type="checkbox"] + label::before, input[type="checkbox"]:checked + label::before {
```

Comment:
Do we need 
```
.clr-form-control-disabled .clr-checkbox-wrapper {
      input[type="checkbox"] + label::before, input[type="checkbox"]:checked + label::before {
```
here?

We could just use:
```
.clr-form-control-disabled {
     input[type="checkbox"] + label::before, input[type="checkbox"]:checked + label::before {
```

no?

Reply:
Only if we unnest the input selectors in the top of this file. This is not an option at the moment, as it would collide with existing forms.

Reply:
But the existing forms dont use `.clr-form-control-disabled` right? I am just saying that we can get rid of `.clr-checkbox-wrapper`.

Reply:
No, because the specificity of the original values that I need to change have `.clr-checkbox-wrapper` in there, because without it new forms would conflict with old forms selectors.